realhf/impl/model/modules/attn.py

```diff
@@ -86,6 +87,7 @@ def __init__(
                 n_q_heads * head_dim,
                 hidden_dim,
                 bias=use_attn_proj_bias,
+                sequence_parallel=constants.sequence_parallel(),
```

Comment:
This is dangerous because the `sequence_parallel` variable may not be set when importing this file. Let's remove this parameter and forcely set `sequence_parallle=constants.sequence_parallel` in the `__init__` method.

Reply:
Fixed.